src/main/java/Application.java

```diff
@@ -0,0 +1,29 @@
+import com.internet.mixshop.lib.Injector;
+import com.internet.mixshop.model.Product;
+import com.internet.mixshop.service.ProductService;
+
+public class Application {
```

Comment:
move this to `com.internet.mixshop` as well

Reply:
done